truss/templates/base.Dockerfile.jinja

```diff
@@ -0,0 +1,103 @@
+ARG PYVERSION={{config.python_version}}
+
+{% if config.model_framework.value == 'huggingface_transformer' %}
+    {% if config.resources.use_gpu %}
+FROM baseten/baseten-huggingface-transformer-server-gpu-base:latest
+    {% else %}
+# TODO(pankaj): Use baseten wrapper, to be consistent with gpu case
+FROM huggingface/transformers-cpu:4.18.0
+    {% endif %}
+{% else %}
+    {% if config.resources.use_gpu %}
+# TODO(pankaj): Replace with baseten/baseten-server-gpu-base-$PYVERSION:latest
+FROM nvidia/cuda:11.2.1-base-ubuntu18.04
+ENV CUDNN_VERSION=8.1.0.77
+ENV CUDA=11.2
+ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
+
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub && \
+    apt-get update && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        cuda-command-line-tools-11-2 \
+        libcublas-11-2 \
+        libcublas-dev-11-2 \
+        libcufft-11-2 \
+        libcurand-11-2 \
+        libcusolver-11-2 \
+        libcusparse-11-2 \
+        libcudnn8=${CUDNN_VERSION}-1+cuda${CUDA} \
+        libgomp1 \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Allow statements and log messages to immediately appear in the Knative logs
+ENV PYTHONUNBUFFERED True
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && \
```

Comment:
Do we need multiple apt-get updates and install here or can we combine with line 18?